lib/activesupport/all/activesupport.rbi

```diff
@@ -548,6 +556,10 @@ class Time
   sig { returns(Time) }
   def beginning_of_day; end
 
+  sig { params(options: T::Hash[Symbol, Integer]).returns(Time) }
+  def advance(options)
```

Comment:
These should be formatted as one line, like `def advance(options); end`

Reply:
Yup. Fixed. Thanks.